stdlib/Dates/src/periods.jl

```diff
@@ -94,6 +94,7 @@ end
 
 (*)(A::Period, B::AbstractArray) = Broadcast.broadcast_preserving_zero_d(*, A, B)
 (*)(A::AbstractArray, B::Period) = Broadcast.broadcast_preserving_zero_d(*, A, B)
+(/)(A::AbstractArray, B::Period) = Broadcast.broadcast_preserving_zero_d(/, A, B)
```

Comment:
Shouldn't actually be deprecated, see https://github.com/JuliaLang/julia/pull/34786 (which tests this).